dependencies/node.py

```diff
@@ -0,0 +1,19 @@
+class node(object):
+    def __init__(self, value, children = []):
+        self.value = value
+        self.children = children
+
+    def __str__(self, level=0):
+        ret = "  -|"*level+repr(self.value)+"\n"
+        for child in self.children:
+            ret += child.__str__(level+1)
+        return ret
+
+    def __repr__(self):
+        return '<tree node representation>'
+
+
+    def saveTree(self, path):
```

Comment:
here is better to use 'with' consitruction instead of implicit open. close, e.g.

```
with open(path, 'w') as f:
    f.write(str(self))
```